doc/how-to/update_upgrade.md

```diff
@@ -0,0 +1,135 @@
+(howto-update-upgrade)=
+# How to update and upgrade
+
+MicroCloud is made of several snaps that are closely coupled with each other.
+The cluster members that are part of the MicroCloud deployment must always run the same version of the snaps.
+This means that when the snaps on one of the cluster members are refreshed, they must also be refreshed on all other cluster members before MicroCloud is operational again.
+See {ref}`howto-update-upgrade-deps` for the recommended order.
+
+Before performing an update or upgrade make sure to backup your data to prevent any data loss in case of failure.
```

Comment:
Should we link to the backup section of `doc/how-to/recover.md`?

Reply:
Fixed. Also updated `doc/how-to/support.md` which also required an update.

Reply:
The link to the `howto-recover` is for troubleshooting details but I think we should link to the specific part that talks about backups in the very sentence we recommend doing backups.

Having a link to the whole recover page for generic troubleshooting details makes sense though.